src/super_gradients/training/pipelines/pipelines.py

```diff
@@ -155,6 +172,8 @@ def _generate_prediction_result_single_batch(self, images: Iterable[np.ndarray])
         # Predict
         with eval_mode(self.model):
             torch_inputs = torch.Tensor(np.array(preprocessed_images)).to(self.device)
```

Comment:
Consider using `torch.from_numpy` as it would skip a data copy (share memory with numpy array).
What type preprocessed_images has that we convert it to `np.array`

Reply:
changed